src/pages/Wallets/WalletDeposit/index.tsx

```diff
@@ -1,20 +1,20 @@
-import React, {useState} from 'react';
+import {useState} from 'react';
 import {useDispatch, useSelector} from 'react-redux';
 import orderBy from 'lodash/orderBy';
 import {mdiRefresh} from '@mdi/js';
 import MdiIcon from '@mdi/react';
-
 import Button, {ButtonColor} from 'components/Button';
 import ExpandableWire from 'components/ExpandableWire';
 import Loader from 'components/Loader';
-import Qr from 'components/Qr';
+import Line from 'components/Line';
 import {createWalletDeposit, getWalletDepositBalance} from 'dispatchers/wallets';
 import {WireType} from 'enums';
 import {useActiveWallet} from 'hooks';
 import {getWires} from 'selectors/state';
 import {AppDispatch, SFC} from 'types';
 import {displayErrorToast} from 'utils/toasts';
 import * as S from './Styles';
+import {colors} from 'styles';
```

Comment:
What I have understood is we have the following imports ordering:

1. React, Redux imports
2. External/third party libraries imports
3. Internal imports
4. Style import

So keeping this order in mind, we will have something like this:

```
import {useState} from 'react';
import {useDispatch, useSelector} from 'react-redux';
import MdiIcon from '@mdi/react';
import {mdiRefresh} from '@mdi/js';
import orderBy from 'lodash/orderBy';

import Button, {ButtonColor} from 'components/Button';
import ExpandableWire from 'components/ExpandableWire';
import Line from 'components/Line';
import Loader from 'components/Loader';
import {AppDispatch, SFC} from 'types';
import {WireType} from 'enums';
import {colors} from 'styles';
import {createWalletDeposit, getWalletDepositBalance} from 'dispatchers/wallets';
import {displayErrorToast} from 'utils/toasts';
import {getWires} from 'selectors/state';
import {useActiveWallet} from 'hooks';

import * as S from './Styles';
```

@buckyroberts Please correct me if I'm wrong/mistaken